### ISSUE TITLE
Atom "trash can" needs two drags to show up

### ISSUE THREAD

**Issue by nikola3244** [gantry/gantry5#1522]
"Drag here to delete" won't show on first drag, on every next drag it works.
Chromium 52.0.2743.116 (64-bit); WP 4.6; Gantry dev-cf863d112 (45 minutes ago).

Problem was there for a while, I just finally found time to report it.

**Comment by w00fz** [comment]
Yeah, I am aware of it and it only happens if you reload the page while you are in `Page Settings`.
This is a little trade off I had to make back when I moved Atoms in Page Settings.

To keep things optimized, I don't initialize any of the Atoms functionalities up until you start interacting with it. Once you are in proximity of the Atoms, the class gets initialized and you can Drag & Drop, edit, add new ones, remove. For some reason the Trash area needs an extra interaction to kick off, since it only happened on refresh when in Page Settings, I figured it was ok.

Thanks for ticketing it though, I'll see if I can fix this edge case.
